client/me/help/help-contact/index.jsx

```diff
@@ -64,6 +65,10 @@ const HelpContact = React.createClass( {
 			this.props.connectHappychat();
 		}
 
+		if ( config.isEnabled( 'help/directly' ) ) {
+			this.props.initializeDirectly();
```

Comment:
I mean doing something like https://github.com/Automattic/wp-calypso/blob/master/client/boot/index.js#L394 so that all code is kept in its own chunk.